File: 백준/Bronze/2566. 최댓값/최댓값.cc
#include <iostream>
#include <sstream>

using namespace std;

int main() {
    int iMax = 0, iRow = 1, iColumn = 1;

    for (int i = 1; i <= 9; i++) {
        string line;
        getline(cin, line);
        stringstream ss(line);
        for (int j = 1; j <= 9; j++) {
            int iNum;
            ss >> iNum;
            if (iNum > iMax) {
                iMax = iNum;
                iRow = i;
                iColumn = j;
            }
        }
    }

    cout << iMax << endl;
    cout << iRow << " " << iColumn << endl;

    return 0;
}

File: 백준/Bronze/2566. 최댓값/최댓값.java
import java.io.*;
import java.util.*;

public class Main {

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int iMax = 0, iRow = 1, iColumn = 1; 

        for (int i = 1; i <= 9; i++) {
            StringTokenizer st = new StringTokenizer(br.readLine());
            for (int j = 1; j<= 9; j++) {
                int iNum = Integer.parseInt(st.nextToken());
                if (iNum > iMax) {
                    iMax = iNum;
                    iRow = i;
                    iColumn = j;
                }
            }
        }
        System.out.println(iMax);
        System.out.println(iRow + " " + iColumn);
    }

}